From 0930e2eaf4654b64a21e85db6d222f6dc140bbe7 Mon Sep 17 00:00:00 2001
From: Sebastian Walderich <sebastian@walderich.de>
Date: Mon, 7 Nov 2011 11:01:19 +0100
Subject: [PATCH] Merged translation strings

Some translated strings where exchanged by similar ones. So a few of them
could be removed and don't have to be translated separately any more.
---
 src/generic/aboutdlgg.cpp |    2 +-
 src/generic/dbgrptg.cpp   |    2 +-
 src/generic/logg.cpp      |    2 +-
 src/generic/prntdlgg.cpp  |    8 ++++----
 src/html/helpwnd.cpp      |    2 +-
 src/propgrid/props.cpp    |    4 ++--
 src/univ/themes/win32.cpp |    2 +-
 src/unix/threadpsx.cpp    |    2 +-
 8 files changed, 12 insertions(+), 12 deletions(-)

diff --git a/src/generic/aboutdlgg.cpp b/src/generic/aboutdlgg.cpp
index fc38c18..ee338f0 100644
--- a/src/generic/aboutdlgg.cpp
+++ b/src/generic/aboutdlgg.cpp
@@ -138,7 +138,7 @@ void wxAboutDialogInfo::SetVersion(const wxString& version,
 
 bool wxGenericAboutDialog::Create(const wxAboutDialogInfo& info, wxWindow* parent)
 {
-    if ( !wxDialog::Create(parent, wxID_ANY, _("About ") + info.GetName(),
+    if ( !wxDialog::Create(parent, wxID_ANY, wxString::Format(_("About %s"), info.GetName()),
                            wxDefaultPosition, wxDefaultSize, wxRESIZE_BORDER|wxDEFAULT_DIALOG_STYLE) )
         return false;
 
diff --git a/src/generic/dbgrptg.cpp b/src/generic/dbgrptg.cpp
index 5d967fc..a6b5561 100644
--- a/src/generic/dbgrptg.cpp
+++ b/src/generic/dbgrptg.cpp
@@ -228,7 +228,7 @@ void wxDumpOpenExternalDlg::OnBrowse(wxCommandEvent& )
                      fname.GetPathWithSep(),
                      fname.GetFullName()
 #ifdef __WXMSW__
-                     , _("Executable files (*.exe)|*.exe|All files (*.*)|*.*||")
+                     , _("Executable files (*.exe)|*.exe|") + wxALL_FILES
 #endif // __WXMSW__
                      );
     if ( dlg.ShowModal() == wxID_OK )
diff --git a/src/generic/logg.cpp b/src/generic/logg.cpp
index 1cb929d..665f53d 100644
--- a/src/generic/logg.cpp
+++ b/src/generic/logg.cpp
@@ -521,7 +521,7 @@ wxLogFrame::wxLogFrame(wxWindow *pParent, wxLogWindow *log, const wxString& szTi
     wxMenuBar *pMenuBar = new wxMenuBar;
     wxMenu *pMenu = new wxMenu;
 #if CAN_SAVE_FILES
-    pMenu->Append(Menu_Save,  _("&Save..."), _("Save log contents to file"));
+    pMenu->Append(Menu_Save,  _("Save &As..."), _("Save log contents to file"));
 #endif // CAN_SAVE_FILES
     pMenu->Append(Menu_Clear, _("C&lear"), _("Clear the log contents"));
     pMenu->AppendSeparator();
diff --git a/src/generic/prntdlgg.cpp b/src/generic/prntdlgg.cpp
index 6a940fe..48733bd 100644
--- a/src/generic/prntdlgg.cpp
+++ b/src/generic/prntdlgg.cpp
@@ -804,7 +804,7 @@ wxComboBox *wxGenericPrintSetupDialog::CreatePaperTypeChoice()
 
     wxComboBox *choice = new wxComboBox( this,
                                          wxPRINTID_PAPERSIZE,
-                                         _("Paper Size"),
+                                         _("Paper size"),
                                          wxDefaultPosition,
                                          wxSize(width, wxDefaultCoord),
                                          n, choices );
@@ -831,7 +831,7 @@ wxGenericPageSetupDialog::wxGenericPageSetupDialog( wxWindow *parent,
                                                     wxPageSetupDialogData* data)
     : wxPageSetupDialogBase( parent,
                 wxID_ANY,
-                _("Page Setup"),
+                _("Page setup"),
                 wxPoint(0,0),
                 wxSize(600, 600),
                 wxDEFAULT_DIALOG_STYLE|wxTAB_TRAVERSAL )
@@ -858,7 +858,7 @@ wxGenericPageSetupDialog::wxGenericPageSetupDialog( wxWindow *parent,
 
     m_paperTypeChoice = new wxComboBox( this,
                                         wxPRINTID_PAPERSIZE,
-                                        _("Paper Size"),
+                                        _("Paper size"),
                                         wxDefaultPosition,
                                         wxSize(300, wxDefaultCoord),
                                         n, choices );
@@ -1060,7 +1060,7 @@ wxComboBox *wxGenericPageSetupDialog::CreatePaperTypeChoice(int *x, int *y)
 
     wxComboBox *choice = new wxComboBox( this,
                                          wxPRINTID_PAPERSIZE,
-                                         _("Paper Size"),
+                                         _("Paper size"),
                                          wxPoint(*x, *y),
                                          wxSize(300, wxDefaultCoord),
                                          n, choices );
diff --git a/src/html/helpwnd.cpp b/src/html/helpwnd.cpp
index d10ee5b..dcd41a5 100644
--- a/src/html/helpwnd.cpp
+++ b/src/html/helpwnd.cpp
@@ -1557,7 +1557,7 @@ void wxHtmlHelpWindow::OnToolbar(wxCommandEvent& event)
 #if wxUSE_LIBMSPACK
                     _("Compressed HTML Help file (*.chm)|*.chm|") +
 #endif
-                    _("All files (*.*)|*");
+                    wxALL_FILES;
                 wxString s = wxFileSelector(_("Open HTML document"),
                                             wxEmptyString,
                                             wxEmptyString,
diff --git a/src/propgrid/props.cpp b/src/propgrid/props.cpp
index 8698b72..bde3b65 100644
--- a/src/propgrid/props.cpp
+++ b/src/propgrid/props.cpp
@@ -1703,7 +1703,7 @@ bool wxPGFileDialogAdapter::DoShowDialog( wxPropertyGrid* propGrid, wxPGProperty
                       property->GetAttribute(wxS("DialogTitle"), _("Choose a file")),
                       property->GetAttribute(wxS("InitialPath"), path),
                       wxEmptyString,
-                      property->GetAttribute(wxPG_FILE_WILDCARD, _("All files (*.*)|*.*")),
+                      property->GetAttribute(wxPG_FILE_WILDCARD, wxALL_FILES),
                       0,
                       wxDefaultPosition );
 
@@ -1732,7 +1732,7 @@ wxFileProperty::wxFileProperty( const wxString& label, const wxString& name,
 {
     m_flags |= wxPG_PROP_SHOW_FULL_FILENAME;
     m_indFilter = -1;
-    SetAttribute( wxPG_FILE_WILDCARD, _("All files (*.*)|*.*") );
+    SetAttribute( wxPG_FILE_WILDCARD, wxALL_FILES);
 
     SetValue(value);
 }
diff --git a/src/univ/themes/win32.cpp b/src/univ/themes/win32.cpp
index 7150c7a..095e95d 100644
--- a/src/univ/themes/win32.cpp
+++ b/src/univ/themes/win32.cpp
@@ -3775,7 +3775,7 @@ void wxWin32FrameInputHandler::PopupSystemMenu(wxTopLevelWindow *window) const
     if ( window->GetWindowStyle() & wxMAXIMIZE_BOX )
         menu.Append(wxID_MAXIMIZE_FRAME , _("Ma&ximize"));
     menu.AppendSeparator();
-    menu.Append(wxID_CLOSE_FRAME, _("Close\tAlt-F4"));
+    menu.Append(wxID_CLOSE_FRAME, _("&Close") + _T("\t") + _("Alt+") + _T("F4"));
 
     if ( window->GetWindowStyle() & wxMAXIMIZE_BOX )
     {
diff --git a/src/unix/threadpsx.cpp b/src/unix/threadpsx.cpp
index 22fefee..46d1857 100644
--- a/src/unix/threadpsx.cpp
+++ b/src/unix/threadpsx.cpp
@@ -815,7 +815,7 @@ void *wxThreadInternal::PthreadStart(wxThread *thread)
     int rc = pthread_setspecific(gs_keySelf, thread);
     if ( rc != 0 )
     {
-        wxLogSysError(rc, _("Cannot start thread: error writing TLS"));
+        wxLogSysError(rc, _("Cannot start thread: error writing TLS."));
 
         return (void *)-1;
     }
-- 
1.7.6.msysgit.0

